tests/openvino/test_quantization.py

```diff
@@ -174,6 +174,11 @@ class OVWeightCompressionTest(unittest.TestCase):
 
     @parameterized.expand(SUPPORTED_ARCHITECTURES_WITH_EXPECTED_8BIT_COMPRESSED_MATMULS)
     def test_automodel_weight_compression(self, model_cls, model_name, expected_pt_int8, expected_ov_int8):
+        import nncf
+
+        if nncf.__version__ == "2.8.0":
```

Comment:
maybe better to use version comparators from optimum instead strings matching?
What if somebody will try to run test with nncf installed from git (it will contain different version, but without fix)

Reply:
Based on changes in the setup.py, I removed this check.